### ISSUE TITLE
[Feature Request] Export AnimationClip to .anim file or similar

### ISSUE THREAD

**Issue by MrDj200** [Er1807/FreezeFrame#4]
Hello,

I recently had the Idea to use VRChat for basic MoCap and was exploring how to do something like that, but then remembered that this mod already does most of the work (basically all of it).
Now I'm wondering if you could look into exporting the `AnimationClip`, which you already have in memory, to a file in either the Unity `.anim` format, or even the `.bvh` format.

In _theory_ it shouldn't be that hard, considering you already got the `AnimationClip` in your [`CreateClip()`](https://github.com/Er1807/FreezeFrame/blob/93d08e829e8a03fa384cc2ffa5dc1ac4428e760a/AnimationModule.cs#L88) Method, but I don't know enough about Unity to say that with 100% certainty.

It would be very nice if you could look into this, since other currently available MoCap software for VR isn't the most intuitive and requires a lot of extra setup.

**Comment by Er1807** [comment]
Yes. It's fairly easy possible. Wanted to anyways make an import export system. So this would most likely be part of that

**Comment by MrDj200** [comment]
Oh nice, that sounds good ^^

**Comment by MrDj200** [comment]
Do you by any chance have an approximate time you're gonna release it?

**Comment by Er1807** [comment]
Can't say. Got some other projects that have priority. Hope it's finished soon but don't know

**Comment by Er1807** [comment]
So I looked at it today and it actually is pretty hard. Reason for it... You can't create a anim file at runtime. Only in the editor. Something possible might be having an intermediate format and then converting that in the editor to the anim file
